tests/integration/tests/test_create_campaign.py

```diff
@@ -0,0 +1,112 @@
+"""
+This is an example automated test to newsletter generator application
+Our automated test will do the following:
+    #Open Qxf2 newsletter generator application
+    #Fill the details of create newsletter section.
+"""
+import os,sys,time
+from typing_extensions import runtime
+sys.path.append(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+from page_objects.PageFactory import PageFactory
+from utils.Option_Parser import Option_Parser
+import conf.create_newsletter_conf as conf
+import pytest
+
+@pytest.mark.GUI
+def test_create_campaign(test_obj):
+
+    "Run the test"
+    #try:
+    if True:
+        #Initalize flags for tests summary
+        expected_pass = 0
+        actual_pass = -1
+        #Create a test object for edit an article
+        test_obj = PageFactory.get_page_object("create newsletter page")
```

Comment:
Change this line to:

PageFactory.get_page_object("create newsletter page", base_url=test_obj.base_url)